src/admin/services/Api.ts

```diff
@@ -85,5 +84,28 @@ const Api = {
 
 	deleteUser:(userId: any): Promise<any> =>
 		sendRequest(`/admin/api/users/${userId}`, 'DELETE'),
+
+	/**
+	 * Get all articles with feedbacks count
+	 * @type {()=>Promise<any>}
+	 */
+	getArticleList: (page?, limit?, sort?, sortOrder?) => {
+		const pagination = setPagination(page, limit, sort, sortOrder);
+		return sendRequest(`/admin/api/articles${pagination}`, 'GET');
+	},
+
+	/**
+	 * Get article by ID
+	 */
+	getArticle: (id) =>
+		sendRequest(`/admin/api/articles/${id}`, 'GET'),
+
+	/**
+	 * Get feedbacks belong to article by ID
+	 */
+	getArticleFeedbacks: (id, page?, limit?, sort?, sortOrder?) => {
+		const pagination = setPagination(page, limit, sort, sortOrder);
```

Comment:
rename setPagination to something like getFormattedPagination or something without set.